hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/http/BaseHttpServer.java

```diff
@@ -194,9 +195,18 @@ public static HttpServer2.Builder newHttpServer2BuilderForOzone(
       final InetSocketAddress httpsAddr, String name) throws IOException {
     HttpConfig.Policy policy = getHttpPolicy(conf);
 
+    String userString = conf.get(OZONE_ADMINISTRATORS, " ");
+    String groupString = conf.get(OZONE_ADMINISTRATORS_GROUPS, "");
+    String userGroupString;
+    if (!groupString.trim().isEmpty()) {
+      userGroupString = userString.trim() + " " + groupString.trim();
+    } else {
+      userGroupString = userString;
+    }
+
     HttpServer2.Builder builder = new HttpServer2.Builder().setName(name)
-        .setConf(conf).setACL(new AccessControlList(conf.get(
-            OZONE_ADMINISTRATORS, " ")));
+        .setConf(conf)
+        .setACL(new AccessControlList(userGroupString));
```

Comment:
Can we skip this string magic by using the `AccessControlList(String user, String groups)` constructor?